spec/route_spec.rb

```diff
@@ -3,24 +3,10 @@
 describe RoutificApi::Route do
   subject(:route) { Factory::ROUTE }
 
-  it "has status" do
-    expect(route.status).to eq(Factory::ROUTE_STATUS)
-  end
-
-  it "has total_travel_time" do
-    expect(route.total_travel_time).to eq(Factory::ROUTE_TOTAL_TRAVEL_TIME)
-  end
-
-  it "has total_idle_time" do
-    expect(route.total_idle_time).to eq(Factory::ROUTE_TOTAL_IDLE_TIME)
-  end
-
-  it "has unserved" do
-    expect(route.unserved).to eq(Factory::ROUTE_UNSERVED)
-  end
-
-  it "has number_of_unserved" do
-    expect(route.number_of_unserved).to eq(Factory::ROUTE_UNSERVED.count)
+  Factory::ROUTE_INPUT.each do |key, value|
+    it "has #{key}" do
+      expect(eval("route.#{key}")).to eq(value)
+    end
   end
 
   describe "#vehicleRoutes" do
```

Comment:
This test checks that calling `add_solution({})` sets an empty hash. That's a nice edge case to test but it would be nice to test the function with a real route too.